code/chart.js

```diff
@@ -1,4 +1,34 @@
+//Font-family for the chart
+Chart.defaults.font.family = 'Source Code Pro, monospace';
+
 //DOM-selector for the canvas 👇
 const ctx = document.getElementById('chart').getContext('2d')
 
 //"Draw" the chart here 👇
+const activateChart = (projects) => {
+    const labels = [
+        'Projects done',
+        'Projects to do'
+      ]
+   
+      const data = {
+        labels: labels,
+        datasets: [{
+
+          data: [projects, 19-projects],
+          label: 'Projects',
+          backgroundColor: ['rgb(244, 132, 95)', 'rgb(220, 220, 220)']
+        }]
+      }
+
+      const config = {
+        type: 'doughnut',
+        data: data,
+        options: {}
+      }
```

Comment:
Doughnuts are some of my favourite charts! I used to work a lot with big data in my old job, and make dashboards for our clients, and I just think doughnuts are superior. Hehe.